docs/API.md

```diff
@@ -31,64 +31,88 @@ MinioClient s3Client = new MinioClient("https://s3.amazonaws.com", "YOUR-ACCESSK
 
```

Comment:
This is not your code/change but I don't see the use of this anchor here:
`<a name="constructors"></a>`

Reply:
Removed .